### ISSUE TITLE
Bug: The `.APK` is considered to be "harmful" by GMS.

### ISSUE THREAD

**Issue by RokeJulianLockhart** [cypherstack/stack_wallet#1045]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When I attempt to install [`fdroid/repo/sw-v2.1.9-arm64-v8a-release.apk`](https://fdroid.stackwallet.com/fdroid/repo/sw-v2.1.9-arm64-v8a-release.apk):

![image](https://github.com/user-attachments/assets/2b39afdd-d221-4c07-9b3f-85faab45d2d0)

...`com.android.vending/com.google.android.finsky.protectdialogs.activity.PlayProtectDialogsActivity` appears:

![image](https://github.com/user-attachments/assets/dc50bd6a-53b0-49e1-85d5-1a6feb86c9bd)

### Expected Behavior

This shouldn't appear. I've never had it appear for any other application.

### Reproduce Steps

https://github.com/user-attachments/assets/d2fe6955-6689-436e-a3b1-8b9202ff72b2

### Environment

- #### Device platform and version:

  Fairphone 5: [^2]

  1. ~~~bash
     #!/usr/bin/env -S bash
     adb shell getprop
     ~~~
  
  1. > ~~~YAML
     > [ro.build.product]: [FP5]
     > ~~~

- #### Operating system and version:

  The OEM distribution of AOSP 14: [^1]

  1. ~~~bash
     #!/usr/bin/env -S bash
     adb shell getprop
     ~~~
  
  1. > ~~~YAML
     > [ro.build.version.release]: [14]
     > [ro.build.display.id]: [FP5.UT2E.B.078.20241105]
     > ~~~

- #### Real device or emulator/simulator:

  Real

[^1]: https://fairphone-android-builds.ams3.digitaloceanspaces.com/FP5/A14/FP5-UT2E-factory.zip
[^2]: https://www.wikidata.org/w/index.php?title=Q122070914&oldid=2278534168#Statements:~:text=All%20entered%20languages-,Statements,-instance%20of

**Comment by lasermind** [comment]
Can confirm, this behaviour is also new in StackDuo version 1.2.4. Did not happen with version 1.2.3.

Google Play Protect wants to scan this app and asks to send it to them.

**Comment by sneurlax** [comment]
This is a result of bundling Monero into Stack Wallet because virus scanners detect Monero mining code as potentially harmful due to its frequent association with malware.  This issue cannot be resolved until (/if) we migrate to another Monero codebase, and even then only until (/unless) malware-scanning services update their scans to target the newer codebase(s), too.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/cypherstack/stack_wallet/issues/1045#issuecomment-2568215550

@sneurlax, consequently, should I close this as unplanned? Irrespective, perhaps the download documentation should explicitly and prominently state your aforementioned?

**Comment by masuteri** [comment]
This is still happening on 2.6.0 with the v8a build from releases

**Comment by sneurlax** [comment]
> This is still happening on 2.6.0 with the v8a build from releases

Yes, we are still using monero's wallet2 which is what's being scanned as potentially harmful due to the mining code which can't be removed.  Again, until we move to eg. a monero-oxide (Rust) core, this will continue to be a problem.  Even then, if the scanners then move to add monero-oxide and other Rust-based Monero cores to their databases, then we'll have the same problem all over again.

This is a common problem for almost all Monero wallets.
